### ISSUE TITLE
Secrets is not compatible with Mac OS 10.6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Zobir/blacktree-secrets#84]
```
It appears most or all preference panes not made specifically for 10.6 do not 
work under it because 
of some new restrictions or something.

They guys making Perian fixed this in the latest version. 
http://trac.perian.org/ticket/413

I hope this helps.
```

Original issue reported on code.google.com by `luc...@gmail.com` on 3 Aug 2009 at 12:05

**Comment by GoogleCodeExporter** [comment]
```
I believe it just needs to be recompiled for 64 bits.
```

Original comment by `daniel.j.jimenez@gmail.com` on 30 Aug 2009 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
Yes, but while this may not be a show stopper having to wait for System 
Preferences to restart just to use a Pane 
it does make it /feel/ broken.
```

Original comment by `sam...@bowles.es` on 30 Aug 2009 at 11:22

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
There, a fix =).
The problem was reported in Console, GC needs to be supported. Fixed a few 
warnings while I was at it.
```

Original comment by `n.oxyde` on 10 Sep 2009 at 11:45

Attachments:
- [Secrets-prefPane-64b.diff](https://storage.googleapis.com/google-code-attachments/blacktree-secrets/issue-84/comment-4/Secrets-prefPane-64b.diff)

**Comment by GoogleCodeExporter** [comment]
```
Can I use this to patch mine to 64 bits?


As a side note, you can avoid having to restart system preferences by having it 
always run in 32-bit mode if you 
want.  Just go to Get Info on the app.
```

Original comment by `everklee...@gmail.com` on 17 Sep 2009 at 11:01

**Comment by GoogleCodeExporter** [comment]
```
The Patch only patches the code itself, it won't patch a compiled version 
already out there.  
```

Original comment by `peelman` on 17 Sep 2009 at 11:28

**Comment by GoogleCodeExporter** [comment]
```
Resolved in 1.0.6
```

Original comment by `peelman` on 21 Sep 2009 at 3:42
- Changed state: **Fixed**
